src/Runner/Baseline/Issue.php

```diff
@@ -112,7 +113,7 @@ public function equals(self $other): bool
         return $this->file() === $other->file() &&
                $this->line() === $other->line() &&
                $this->hash() === $other->hash() &&
-               $this->description() === $other->description();
+               ($this->description() === $other->description() || preg_match('{' . $this->description() . '}', $other->description) !== false);
```

Comment:
I'm actually wondering if we should use regex, or a simple `str_contains()`, WDYT?